app/javascript/analytics.js

```diff
@@ -1,76 +1,76 @@
-$(function() {
+document.addEventListener('turbolinks:load', () => {
```

Comment:
You are definitely correct.  I extracted the GA setup stuff so that the analytics reporting will still fire every page load now but not try and set it up multiple times.